plugins/blur/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ set(SOURCES
     blur.cpp)
 
 if (APPLE)
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -stdlib=libc++")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -stdlib=libc++")
```

Comment:
>  Out of curiosity, shouldn't the changes inside (APPLE) clauses only affect compiling on OSX?

In fact I don't understand why it's not something like:

`
if(UNIX AND NOT APPLE)
    set(LINUX TRUE)
endif()

if(LINUX)
...
endif()
`